nerfstudio/data/datamanagers/base_datamanager.py

```diff
@@ -461,7 +461,7 @@ def create_eval_dataset(self) -> TDataset:
 
     def _get_pixel_sampler(self, dataset: TDataset, num_rays_per_batch: int) -> PixelSampler:
         """Infer pixel sampler to use."""
-        if self.config.patch_size > 1:
+        if self.config.patch_size > 1 and isinstance(self.config.pixel_sampler, PixelSamplerConfig):
```

Comment:
why would pixel_sampler ever not be an instance of PixelSamplerConfig?

Reply:
Right, it should be `and type(self.config.pixel_sampler) is PixelSamplerConfig` since then it will return False if `self.config.pixel_sampler` is a subclass.